src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -684,6 +684,11 @@ impl_install_tree (RpmOstreeTreeComposeContext *self,
                    GCancellable    *cancellable,
                    GError         **error)
 {
+  /* Set this early here, so we only have to set it one more time in the
+   * complete exit path too.
+   */
+  *out_changed = FALSE;
+
```

Comment:
so setting this here means that if we return before line 719 then this will already be set for the caller, right?

Reply:
Yup, exactly!